components/MainNavBarItem.vue

```diff
@@ -24,14 +24,22 @@
    computed: {
      isActive : function(){
        return this.url === this.$route.path;
+     },
+     isSubpageActive: function() {
+       return this.subpages.some(page => page.url === this.$route.path);
      }
    }
  }
 </script>
 
 <style>
 
- a.menuActive {
+ .menuActiveBar {
+   color: #000;
+   text-decoration-line: underline;
```

Comment:
Just a personal preference, but I don't think we need the underline. Think it looks better without. But happy to stick with the underline if you have been asked to do this, or you would really prefer to keep.

Reply:
Ewan thought was clearer with underline. But is something can change in future.